system/doc/efficiency_guide/retired_myths.xml

```diff
@@ -60,4 +60,17 @@
     That leads us to the myth that tail-recursive functions are faster
     than body-recursive functions.</p>
   </section>
+
+  <section>
+    <title>Myth: List subtraction ("--" operator) is slow</title>
+
+    <p>List subtraction used to have a run-time complexity proportional to the
+      product of the length of its operands, which was extremely slow on long
+      lists.</p>
+
+    <p>These days the run-time complexity is "n log n" and the operation will
+      complete quickly even on very long lists. In fact, it's faster and uses
```

Comment:
"it's": OTP's documentation guidelines says that contraction should be avoided.

Reply:
Thanks, fixed!